com.unity.ml-agents/Runtime/Inference/BarracudaModelParamLoader.cs

```diff
@@ -599,24 +635,22 @@ static IEnumerable<string> CheckOutputTensorPresence(Model model, int memory)
             Model model,
             BrainParameters brainParameters,
             ActuatorComponent[] actuatorComponents,
-            ModelActionType isContinuous,
+            ModelActionType modelActionType,
             int modelContinuousActionSize, int modelSumDiscreteBranchSizes)
         {
             var failedModelChecks = new List<string>();
-            if (isContinuous == ModelActionType.Unknown)
-            {
-                failedModelChecks.Add("Cannot infer type of Control from the provided model.");
-                return failedModelChecks;
-            }
-            if (isContinuous == ModelActionType.Continuous &&
+
+            // TODO: Update this check after intergrating ActionSpec into BrainParameters
```

Comment:
If anything, we're going to be deprecating SpaceType, not adding Hybrid to it. 

I'm not sure how to keep these checks meaningful, since we can have a hybrid action but the Agent's BrainParameters can still be all continuous or all discrete.

Reply:
In that case, maybe checking all the continuous/discrete sizes and branches to match between BrainParameters and the model is sufficient?
and just delete these checks

Reply:
I guess you can check that if `brainParameters.VectorActionSpaceType == SpaceType.Continuous` that the number of actions is <= the total number of model continuous actions (and similar for discrete)